src/app/store/Category/Category.reducer.js

```diff
@@ -9,6 +10,7 @@ const initialState = {
     items: [],
     totalItems: 0,
     category: {},
```

Comment:
This one looks unused.

Reply:
It is used in case UPDATE_CURRENT_CATEGORY: